### PR TITLE
KeyframeTrack: Improve docs and subtype ctors.

### PR DESCRIPTION
**Pull request by trusktr**
**Description**

Adds a few small details to docs for KeyframeTrack and its subtypes. Also expresses the lack of `interpolation` parameters for Boolean and String keyframe tracks in their class `constructor`. 

*This contribution is funded by [Lume - 3D HTML elements](https://lume.io)*

### CODE REVIEW

File: docs/api/en/animation/tracks/BooleanKeyframeTrack.html [hunk 1]
```diff
@@ -23,14 +23,18 @@ <h3>
 			[page:Array times] - (required) array of keyframe times.<br />
 			[page:Array values] - values for the keyframes at the times specified.<br />
 		</p>
+		<p>
+			This keyframe track type has no interpolation parameter because the
+			interpolation is always [page:Animation InterpolateDiscrete].
```
Comment (Mugen87) [inline docs/api/en/animation/tracks/BooleanKeyframeTrack.html:28]: If we want to make the API documentation technically correct, `BooleanKeyframeTrack` should also document the `interpolation` ctor parameter. The class does not overwrite the ctor so it inherits the same signature as `KeyframeTrack`.

We can state in the description which interpolation types are allowed.

File: src/animation/tracks/StringKeyframeTrack.js [hunk 2]
```diff
@@ -4,7 +4,16 @@ import { KeyframeTrack } from '../KeyframeTrack.js';
 /**
  * A Track that interpolates Strings
  */
-class StringKeyframeTrack extends KeyframeTrack {}
+class StringKeyframeTrack extends KeyframeTrack {
+
+	// No interpolation parameter because only InterpolateDiscrete makes sense.
```
Comment (Mugen87) [inline src/animation/tracks/StringKeyframeTrack.js:9]: Instead of using the term "makes sense", can you please switch to "is valid"?

Comment (trusktr) [reply src/animation/tracks/StringKeyframeTrack.js:9]: updated this, also updated tests.

**Comment by trusktr** [comment]
e2e snapshot tests failed on the last commit for some reason, but in that last commit I didn't change code, I only removed the unused parts of the unit tests. Seems like a fluke.

I can't get the e2e tests to run locally either:

```
$ npm run test-e2e

> three@0.164.1 test-e2e
> node test/e2e/puppeteer.js

file:///Users/trusktr/src/mrdoob+three.js/node_modules/@puppeteer/browsers/lib/esm/launch.js:301
                reject(new Error([
                       ^

Error: Failed to launch the browser process!


TROUBLESHOOTING: https://pptr.dev/troubleshooting

    at Interface.onClose (file:///Users/trusktr/src/mrdoob+three.js/node_modules/@puppeteer/browsers/lib/esm/launch.js:301:24)
    at Interface.emit (node:events:526:35)
    at Interface.close (node:internal/readline/interface:527:10)
    at Socket.onend (node:internal/readline/interface:253:10)
    at Socket.emit (node:events:526:35)
    at endReadableNT (node:internal/streams/readable:1376:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

Node.js v20.6.1
```

**Review by Mugen87 — APPROVED** [review]

**Pull request merged**